### ISSUE TITLE
SteamVR interface not found (105)

### ISSUE THREAD

**Issue by yotambahat** [ValveSoftware/steamvr_unity_plugin#908]
I have a problem with the SteamVR plugin. I created a new project, imported the latest SteamVR plugin (2.7.2 sdk 1.14.15)
When running unity, I don't see the controllers in the 'Game' window and they don't work (none of the commands/buttons are working). The only hint I have that there is a problem is the message I get (SteamVR Interface Not Found (105)) I tried to look for a solution on the net, but couldn't find any solution. Please help/advise.

using Windows10
Unity version 2018.4.20

**Comment by zite** [comment]
Sorry you're having trouble. What version of SteamVR are you running?

**Comment by yotambahat** [comment]
(2.7.2 sdk 1.14.15)

**Comment by yotambahat** [comment]
I tried it on two different machines with two different head sets...

**Comment by Shii2** [comment]
Same issue as I reported here: https://github.com/ValveSoftware/steamvr_unity_plugin/issues/901

**Comment by ict-ryahata** [comment]
I have the same issue as @Shii2 and probably the same as OP.

I get the same error in my test scene as well as Interaction_Example not working (displays the VR simulator view).

Using Unity 2019.4.19f1 in Legacy VR mode, Unity SteamVR plugin ver 2.7.2, SteamVR 1.15.19.

**Comment by gwen764** [comment]
As stated above, I have exactly the same issue. 

Unity 2019.4.11f1 (also tried it on 2019.4.20f1 and 2018.4.31f1) 
SteamVR (2.7.2 sdk 1.14.15) 
Legacy VR Mode

When the headset is fully connected and working, I generated the SteamVR Input following default action sets, but the controlls didn't bind (as I can see in Live Preview on top - Not Bound | Inactive ).

![image](https://user-images.githubusercontent.com/79167218/108112596-9c9ef300-7096-11eb-8ed7-550eb77c9de1.png)

-  When using the **[CameraRig]** prefab and running game, I can look around through the headset, but no controllers are visible and no controlls work.

- When using the **Player** prefab and running game, the view jumps into simulation mode, and doesn't even show up in headset and I can't rotate the view of the camera with it, no controllers are visible and no controlls work.

![image](https://user-images.githubusercontent.com/79167218/108112931-1800a480-7097-11eb-911d-7d2d22ca95c5.png)

Both stated scenarios above reproduce this warning 2 times

```
[SteamVR] Interface Not Found (105)
UnityEngine.Debug:LogWarning(Object)
Valve.VR.SteamVR:ReportError(EVRInitError) (at Assets/SteamVR/Scripts/SteamVR.cs:244)
Valve.VR.SteamVR:CreateInstance() (at Assets/SteamVR/Scripts/SteamVR.cs:182)
Valve.VR.SteamVR:Initialize(Boolean) (at Assets/SteamVR/Scripts/SteamVR.cs:103)
Valve.VR.SteamVR_Behaviour_Pose:OnEnable() (at Assets/SteamVR/Input/SteamVR_Behaviour_Pose.cs:88)
```
In addition, the Player prefab outputs one more similiar warning

```
[SteamVR] Interface Not Found (105)
UnityEngine.Debug:LogWarning(Object)
Valve.VR.SteamVR:ReportError(EVRInitError) (at Assets/SteamVR/Scripts/SteamVR.cs:244)
Valve.VR.SteamVR:CreateInstance() (at Assets/SteamVR/Scripts/SteamVR.cs:182)
Valve.VR.SteamVR:Initialize(Boolean) (at Assets/SteamVR/Scripts/SteamVR.cs:103)
Valve.VR.SteamVR_Behaviour:InitializeSteamVR(Boolean) (at Assets/SteamVR/Scripts/SteamVR_Behaviour.cs:119)
Valve.VR.SteamVR_Behaviour:Awake() (at Assets/SteamVR/Scripts/SteamVR_Behaviour.cs:100)

```

**Comment by D3rpDoge** [comment]
I have the same issue but so far no solution to fixing it

**Comment by D3rpDoge** [comment]
Ok so I messed around with versions of the unity editor and found a way to fix it, the only way I was able to fix it was by changing the editor version to the latest (2020.2.4f1)

**Comment by zite** [comment]
This should be fixed now as of 2.7.3. Sorry about the wait.
